internal/auth/auth_test.go

```diff
@@ -129,9 +129,7 @@ func TestVerifyAccount(t *testing.T) {
 				return tt.context.account, tt.context.err
 			}
 
-			_, err := VerifyAccount(tt.args.username, tt.args.password)
-
-			if err != tt.result.err {
+			if _, err := VerifyAccount(tt.args.username, tt.args.password); err != tt.result.err {
```

Comment:
nit: You don't have to change this but personally I find this exchange of horizontal complexity for vertical complexity to be more difficult to read

Reply:
I've reverted this